package.json

```diff
@@ -91,6 +91,7 @@
     "lodash": "^4.17.15"
   },
   "peerDependencies": {
-    "react": "^17.0.0"
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1"
```

Comment:
Billy, I did some more research and I was wrong about needing to add react-dom here. We'll want to remove it from the peer dependencies list please. Sorry :(